packages/projection-typeorm/src/entity/StakeKey.entity.ts

```diff
@@ -0,0 +1,8 @@
+import { Ed25519KeyHashHex } from '@cardano-sdk/crypto';
+import { Entity, PrimaryColumn } from 'typeorm';
+
+@Entity('stake_key')
+export class StakeKeyEntity {
```

Comment:
I would use `BlockCascadeRelationOptions` for all the entries; this will reduce the code to handle rollback to just a single `DELETE` statement.

Reply:
That doesn't work well for all use cases. For example, this table represents "an active set of stake keys", valid for latest 'block'. We're not projecting the certificates directly, so there is no association of specific stake key with a specific block. Sure we could change the model to fit `BlockCascadeRelationOptions`, but the "state at latest block" type of projections are really nice when it's possible to do.

Reply:
Initially I didn't got the meaning of the table. Thank you for the explanation.